### PR TITLE
Add Create and Attach volumes endpoint docs

### PR DESCRIPTION
**Pull request by RobertDeRose**

### CODE REVIEW

File: docs/src/data/endpoints/volumes.yaml [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+            description: >
+              The size in GBs that you wish to make your new volume.
+              Defaults to 20 GB, can be as large as 1024 GB (1 TB)
+            type: Integer
+          linode_id:
+            optional: true
+            description: >
+              An id to a Linode you'd like this volume to be attached to after
+              creation. Requires an additional scope of `linode:modify` and you
+              must have permission to access that give Linode.
```
Comment (yoshizzle) [inline docs/src/data/endpoints/volumes.yaml]: misspelled "give"

File: docs/src/data/endpoints/volumes.yaml [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+            type: String
+          region:
+            description: >
+              Which region the new volume should be created in
+            type: Datacenter
+          size:
+            optional: true
+            description: >
+              The size in GBs that you wish to make your new volume.
+              Defaults to 20 GB, can be as large as 1024 GB (1 TB)
```
Comment (yoshizzle) [inline docs/src/data/endpoints/volumes.yaml]: needs a period

File: docs/src/data/endpoints/volumes.yaml [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+      POST:
+        oauth: volume:modify
+        description: >
+          Attach a Block Storage Volume to a Linode
+        params:
+          linode_id:
+            description: >
+              An id to a Linode you'd like this volume to be attached to.
+              Requires an additional scope of `linode:modify` and you
+              must have permission to access that give Linode.
```
Comment (yoshizzle) [inline docs/src/data/endpoints/volumes.yaml]: "give" misspelled again

File: docs/src/data/endpoints/volumes.yaml [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+            description: >
+              An id to a Linode you'd like this volume to be attached to after
+              creation. Requires an additional scope of `linode:modify` and you
+              must have permission to access that give Linode.
+            type: Linode
+          config_id:
+            optional: true
+            description: >
+              An id to a Linode configuration profile that should be used when
+              attaching to a Linode that has more than one configuration profile.
```
Comment (yoshizzle) [inline docs/src/data/endpoints/volumes.yaml]: This description doesn't read perfectly clear to me, but I don't really have a solid suggestion for replacement at the moment.

File: docs/src/data/endpoints/volumes.yaml [hunk 5]
```diff
@@ -0,0 +1,88 @@
...
+            description: >
+              An id to a Linode you'd like this volume to be attached to.
+              Requires an additional scope of `linode:modify` and you
+              must have permission to access that give Linode.
+            type: Linode
+          config_id:
+            optional: true
+            description: >
+              An id to a Linode configuration profile that should be used when
+              attaching to a Linode that has more than one configuration profile.
```
Comment (yoshizzle) [inline docs/src/data/endpoints/volumes.yaml]: Same as my other comment above.

File: docs/src/data/endpoints/volumes.yaml (hunk 3 above)
Comment (RobertDeRose) [inline docs/src/data/endpoints/volumes.yaml]: It should be `given` you mean, yeah thanks

File: docs/src/data/endpoints/volumes.yaml [hunk 6]
```diff
@@ -16,12 +16,12 @@ endpoints:
...
             description: >
-              A unique label to identify your new volume with
+              A unique label to identify your new volume with.
             type: String
           region:
             description: >
-              Which region the new volume should be created in
-            type: Datacenter
+              Which region the new volume should be created in.
+            type: Integer
```
Comment (RobertDeRose) [inline docs/src/data/endpoints/volumes.yaml]: Datacenters aren't Integers, they are strings, because we use slugs, no?

Comment (eatonphil) [inline docs/src/data/endpoints/volumes.yaml]: Ah, crap. Will make a patch. Thank you

**Pull request merged**